web_app/models/corrosion_model.py

```diff
@@ -15,6 +15,8 @@ class CorrosionModel(Model):
         model_name (str): The name of the corrosion model.
     """
 
+    MODEL_COORDINATES = None # Initialize the coordinates associated with the model to be None
```

Comment:
This should be a member variable. 
Since super().__init()__ is called first you can use the parent class member variable.

Reply:
will move